test/node/uuid.test.ts

```diff
@@ -62,51 +51,23 @@ describe('UUID', () => {
     expect(uuid2.toString()).to.equal(LOWERCASE_DASH_SEPARATED_UUID_STRING);
   });
 
-  /**
-   * @ignore
-   */
   it('should correctly create UUID from UUID (copying existing buffer)', () => {
     const org = new UUID();
     const copy = new UUID(org);
     expect(org.id).to.not.equal(copy.id);
     expect(org.id).to.deep.equal(copy.id);
   });
 
-  /**
-   * @ignore
-   */
-  it('should throw if passed invalid 36-char uuid hex string', () => {
-    expect(() => new UUID(LOWERCASE_DASH_SEPARATED_UUID_STRING)).to.not.throw();
-    expect(() => new UUID('aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa')).to.throw(BSONError);
-    // Note: The version is missing here ^
-  });
-
-  /**
-   * @ignore
-   */
   it('should throw if passed unsupported argument', () => {
     expect(() => new UUID(LOWERCASE_DASH_SEPARATED_UUID_STRING)).to.not.throw();
     expect(() => new UUID({})).to.throw(BSONError);
   });
 
-  /**
-   * @ignore
-   */
   it('should correctly check if a buffer isValid', () => {
     const validBuffer = Buffer.from(UPPERCASE_VALUES_ONLY_UUID_STRING, 'hex');
-    const invalidBuffer1 = Buffer.from('aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa', 'hex');
-    const invalidBuffer2 = Buffer.alloc(16);
-
-    expect(validBuffer.length).to.equal(invalidBuffer1.length);
-    expect(validBuffer.length).to.equal(invalidBuffer2.length);
-    expect(UUID.isValid(invalidBuffer1)).to.be.false;
-    expect(UUID.isValid(invalidBuffer2)).to.be.false;
```

Comment:
we're missing a lot of coverage here (we're now only testing the success case).  Should we add tests for the expected failure cases?  

I'm asking because you specifically removed the only two error tests we had.

Reply:
Added isValid coverage, fixed the instanceof usage